src/wrapper/objects/global_ref.rs

```diff
@@ -41,8 +41,11 @@ impl<'a> From<&'a GlobalRef> for JObject<'a> {
 }
 
 impl GlobalRef {
-    /// Creates a new global reference. This assumes that `NewGlobalRef`
-    /// has already been called.
+    /// Creates a new global reference.
```

Comment:
That's a little misleading summary — it takes a global reference as `obj` (rename to `global_ref`/`raw_global_ref`/`inner_global_ref`?), and creates a wrapper for it (`GlobalRef`). It does not create new global references (in JNI terms).

Can it be/does it makes sense to make it pub (crate)? It seems no one shall normally use it from the outside.

Reply:
Made it pub(crate) -> deleted the integration test for testng incorrect usage of JNI.